src/routes/index.js

```diff
@@ -13,5 +13,6 @@ router.get('/ping', (req, res) =>
 	res.json({ message: 'Hello! You have found the zc_plugin_reminder api' })
 )
 router.route('/reminders').get(reminderController.create)
+router.route('/reminders/:id').delete(reminderController.deleteById)
```

Comment:
What if they don't add the `ID`, how are we checking for this? Enforce error checks

Reply:
I'll fix this right away.